lib/apiForSheets.js

```diff
@@ -70,8 +70,15 @@ module.exports.generateOutput = async function(rows, prNumber, prDetails) {
     }
 
     // If the PR does not have a CLA label and the author has signed
-    // the CLA, no action is required.
+    // the CLA, welcome the author.
     if (hasUserSignedCla === true) {
+      var commentParams = context.repo({
+        number: prNumber,
+        body: 'Hi! @' + userName + '. Welcome to Oppia! ' +
```

Comment:
I don't think you need to thank them for signing the CLA every time they make a PR ...?

Reply:
Should we add this (to put up a project label when a review is required) in the checklist?

Reply:
Yes, I think so.